### PR TITLE
Added Internal Barcode Scanner Functionality!

### PR DESCRIPTION
**Pull request by MinickLS**

### CODE REVIEW

**Comment by kaysabelle** [comment]
This looks good to me. I can merge with approval of 1 other reviewer.

File: radlibDemoApp/www/index.html [hunk 1]
```diff
@@ -48,7 +48,7 @@ <h1 class="topcoat-navigation-bar__title">RadLib Demo</h1>
                         </a>
                     </li>
                     <li class="topcoat-list__item side-nav__list__item is-active">
-                        <a href="#" class="side-nav__button" onclick="alert('Â© 2015 Team RadLib')">
+                        <a href="#" class="side-nav__button" onclick="alert('© 2015 Team RadLib')">
                             <p>About</p>
```
Comment (jeslin) [inline radlibDemoApp/www/index.html:52]: So fancy

**Comment by kaysabelle** [comment]
#mergefest2k15

**Comment by jeslin** [comment]
CONGRATS ON BEING THE FIRST TO BE MERGED

**Pull request merged**